plutus-ledger-api/src/PlutusLedgerApi/V1/Time.hs

```diff
@@ -35,7 +35,8 @@ newtype DiffMilliSeconds = DiffMilliSeconds Integer
 
 makeLift ''DiffMilliSeconds
 
--- | POSIX time is measured as the number of milliseconds since 1970-01-01T00:00:00Z
+-- | POSIX time is measured as the number of milliseconds since 1970-01-01T00:00:00Z.
+-- Must be non-negative.
```

Comment:
Where does it say that it cannot be negative? Especially since POSIXTime is used as an `Interval` why the lower bound cannot be negative? @michaelpj

Reply:
POSIXTime is seconds in c (int32 or int64) or in Haskell internally (Integer that represents picoseconds)

Reply:
I think we've confirmed that POSIXTime is in milliseconds. It was listed in one of the issues as an invariant that we should document. I thought it made sense from its existing Haddock because it's definitely after 1970?

Reply:
Yes confimed to be milliseconds. I thought perhaps to leave a note that this is not the same as Haskell's `Data.Time.Clock.POSIX.POSIXTime`

Reply:
OK I'll remove it since we're not sure.